examples/bvh_driver/bvh_driver.cpp

```diff
@@ -267,21 +274,17 @@ class CmdLineArgs
       _argv.push_back(new char[s.size() + 1]);
       std::strcpy(_argv.back(), s.c_str());
     }
+    _argv_copy = _argv;
   }
 
   ~CmdLineArgs()
   {
-    // Benchmark removes its own arguments from the command line arguments.
-    // This means, that by virtue of returning references to internal data
-    // members in argc() and argv() function, it will necessarily modify the
-    // members. It will decrease _argc, and "reduce" _argv data. This, however,
-    // will result in _argv having the original size, but the data beyond first
-    // _argc values is garbage, and would have been already released.
-    for (int i = 0; i < _argc; ++i)
+    for (auto const &p : _argv_copy)
```

Comment:
Reference was unnecessary in the 1st place.  I find the `const` qualifier disturbing.